static/js/auth.ts

```diff
@@ -278,34 +285,6 @@ $ ('.auth input').get ().map (function (el) {
 });
 
 // We use GET /profile to see if we're logged in since we use HTTP only cookies and cannot check from javascript.
-$.ajax ({type: 'GET', url: '/profile'}).done (function (response) {
-  if (['/signup', '/login'].indexOf (window.location.pathname) !== -1) auth.redirect ('my-profile');
-
-  auth.profile = response;
```

Comment:
This were was a load-bearing statement.

I agree it's not fantastic, but fetching `/profile` using AJAX and then copying the response onto `auth.profile` is how the Save button knows you are logged in.

So either you need to put this back, or come up with a new way of setting the same property (or checking for the logged-in condition differently).